examples/README.md

```diff
@@ -1,26 +1,29 @@
 ## Labelbox SDK Examples
 
 * Learn how to use the SDK by following along
+* Run in google colab, view the notebooks on github, or clone the repo and run locally
 
-Structure:
+Tutorials:
 
-1. basics
-    * [Fundamentals](basics/basics.ipynb)
+1. [Basics](basics)
```

Comment:
I think this is a much clearer way to format  https://github.com/googleapis/nodejs-storage#samples

Like a nice table or something like that, I think it will make the docs much more presentable and nice